frontend/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -7,15 +7,25 @@ import { useRiders } from '../../context/RidersContext';
 import { Rider } from '../../types/index';
 import { useHistory } from 'react-router-dom';
 import { ToastStatus, useToast } from '../../context/toastContext';
+import { useEmployees } from '../../context/EmployeesContext';
 
 type ConfirmationProps = {
   open: boolean;
   rider?: Rider;
   onClose: () => void;
+  employeeId?: string;
```

Comment:
I think the Props should be more descriptive. For example, 
`open, isRider: bool, onClose, id, role` (id is employee id or student id depending on the isRider bool) or
`open, isRider: bool, onClose, user` (here user is Admin | Driver type or Rider Type depending on isRider)

I think we are only using the rider.id, so we don't need the  entire rider object